engine/flink/management/src/main/scala/pl/touk/nussknacker/engine/management/FlinkProcessManager.scala

```diff
@@ -65,7 +65,8 @@ abstract class FlinkProcessManager(modelData: ModelData, shouldVerifyBeforeDeplo
 
   override def cancel(processName: ProcessName, user: User): Future[Unit] = {
     findJobStatus(processName).flatMap {
-      case Some(ProcessState(Some(deploymentId), _, _, actions, _, _, _, _, _, _)) if actions.contains(ProcessActionType.Cancel) =>
+      // Flink does not allow to cancel 'completed' job
```

Comment:
Maybe not 'completed' but 'not runnint'?

Reply:
Ok. 'completed' was supposed to refer to `Completed Job List` in Flink UI Console